src/Playwright/Transport/Connection.cs

```diff
@@ -461,5 +453,14 @@ private void Dispose(bool disposing)
             {
             }
         }
+
+        [Conditional("DEBUG")]
+        private void TraceMessage(string message)
+        {
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("DEBUGPWD")))
```

Comment:
Is `DEBUGPWD` some kind of standard debug env in dotnet? If not, why don't we use the same `DEBUG=pw*` as we do everywhere?

Reply:
No, it's purely internal, and it's meant purely for developing the dotnet port. It's the same approach as in [Python](https://github.com/microsoft/playwright-python/blob/21b7d1ca59dce3b22c17d348bce54b7ae0e0780d/playwright/_impl/_transport.py#L72).

The PR encompasses two major changes.

1) The `stderr` redirect now happens in a single line (see link above)
2) The internal _tracing_ that's potentially useful for the debugging of the port itself, happens with the `TraceMessage` method and is not compiled into the production builds.

Reply:
To add some context: we *used* to add a bunch of our (=dotnet specific) internal logging to the same output produces by the driver, when using `DEBUG=pw*`. This no longer happens. So, whatever you set in your `DEBUG`, you get the output into `stderr`, displayed either in NUnit's Test Window, or in the Console. 

For our internal stuff, because I still want it there, I moved it to the right place, which is to use the `System.Diagnostics` mechanism.